libkbfs/mdserver_remote.go

```diff
@@ -351,6 +354,8 @@ func (md *MDServerRemote) OnDisconnected(ctx context.Context,
 		md.log.CWarningf(ctx, "MDServerRemote is disconnected")
 		md.config.Reporter().Notify(ctx,
 			connectionNotification(connectionStatusDisconnected))
+		md.reconnectBackoff.Start(reconnectBackoffWindow)
+		defer md.reconnectBackoff.Wait()
```

Comment:
It's only called in the reconnect routine I think. But yeah I agree it's a good idea to add a method to the interface. I'll try to refactor this a little bit, which also has the benefit of making it more generally useful. Thanks!